Agendas/ST-vesting.md

```diff
@@ -0,0 +1,73 @@
+# TABLE OF CONTENTS
+1 [Security Target Introduction](#1-security-targets-introduction)<br>
+1.1 Security Target Reference <br>
+1.2 TOE Reference <br>
+1.3 Terminology and Acronyms<br>
+1.4 Product Overview<br>
+1.5 TOE Overview <br>
+1.6 TOE Description<br>
+2 Conformance Claims<br>
+3 TOE Security Problem Definition<br>
+3.1 Assumption<br>
+3.2 Threats<br>
+3.3 Organizational Security Policies<br>
+4 Security Objectives<br>
+4.1 Security Objectives for the TOE<br>
+4.2 Security Objectives for the Operational Environment<br>
+5 Extended Components<br>
+5.1 Extended Security Functional Requirement (SFR)<br>
+5.2 Extended Security Assurance Requirement (SAR)<br>
+6 TOE Security Requirements<br>
+6.1 Conventions<br>
+6.2 Security Functional Requirements (SFR)<br>
+6.3 Security Assurance Requirements<br>
+7 TOE Summary Specifications<br>
+7.1 User Data Protection<br>
+7.2 Identification and Authentication<br>
+7.3 Security Management<br>
+7.4 TOE Access<br>
+8 Rationale<br>
+8.1 Protection Profile Conformance Claim Rationale<br>
+8.2 Security Objectives Rationale<br>
+8.3 Extended Security Functional Requirement Rationale<br>
+8.4 Extended Security Assurance Requirement Rationale<br>
+8.5 Security Functional Requirements Rationale<br>
+
+# 1. Security Targets Introduction
+
+1.1 Security Target Reference <br>
+
+TODO
+
+1.2 TOE Reference <br>
+
+TODO
+
+1.3 Terminology and Acronyms<br>
+
+TODO
+
+1.4 Product Overview<br>
+
+product is a vesting service <br>
+users is allowed to test the smart contract by submitting transactions to the blockchain emulator.<br>
+TOE for this evaluation will only focus on the API engine that used by ?? <br>
+The product consists of the following components: <br>
+the validator script, the transaction builder, testing harness.
+
+1.5 TOE Overview <br>
+
+TOE Overview summarizes the usage and major security features of the TOE. TOE Overview provides
+context for the evaluated TOE by identifying the TOE type, describing the product, and defining the
+specific evaluated configuration. <br>
+
+1.5.X TODO: Usage and features, Type, Non-TOE hardware/firmware/software required by the TOE
```

Comment:
There's an interesting discussion here. I don't think we need to worry about hardware/firmware in our industry because we basically eliminate hardware errors by having the computation distributed and checked across the nodes (which also means we have no idea which hardware will run the computation).

That being said, there could be a discussion if, for example, an attacker could access the HW where a private key is stored or if the HW where the key is stored is vulnerable to some attack to extract the private key.

But do we want to consider that in our ST?

Reply:
Indeed.

Do you mean security objectives?

![image](https://github.com/input-output-hk/Certification-working-group/assets/36756030/ef320518-346d-4891-ac06-99f28e992eed)

https://www.commoncriteriaportal.org/files/epfiles/FNS%20-%20Guardian%20-%20CCS%20Blockchain%20Secure%20Authentication%20(BSA)%20Security%20Target%20-%201.1.pdf